### DISCUSSION TITLE
关于字幕获取功能的问题

### DISCUSSION THREAD

**Discussion by Muyi030** [Nemo2011/bilibili-api#413] [Q&A] [answered]
现在字幕获取功能还能用吗，想调用接口获取视频中自动生成的字幕，试了几个视频，返回来的sub都是为空。望解答，谢谢
！
`v = video.Video(bvid='BV1xt411o7Xu')
subtitle = sync(v.get_subtitle(1));`

**Comment by z0z0r4** [comment, marked as answer]
加 credential
